src/transformState.js

```diff
@@ -5,7 +5,21 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
```

Comment:
destructurize action
```suggestion
  for (const {type, ....} of actions) {
```